privaterelay/locales/en-US/app.ftl

```diff
@@ -212,11 +213,27 @@ banner-choose-subdomain-label = Your domain is:
 # $subdomain (url) - User-set subdomain
 banner-choose-subdomain-label = You can make up any address @{ $subdomain }
 
-remaining-aliases-cta = Buy unlimited aliases
+## Error Messages
+
+# Variables:
+#   $number (string) - Maximum number of aliases a user can make on a free account. 
+error-premium-set-make-aliases = You must be a premium subscriber to make more than { $number } aliases
+error-premium-cannot-change-subdomain = You cannot change your subdomain
+error-premium-set-subdomain = You must be a premium subscriber to set a subdomain
+error-premium-set-create-subdomain = You must be a premium subscriber to create subdomain aliases
+error-subdomain-not-created = Subdomain could not be created, try something else
+error-subdomain-email-not-created = Email address with subdomain could not be created, try something else
+error-subdomain-select = You must select a subdomain before creating email address with subdomain
```

Comment:
Should it be `creating _an_ email address with subdomain?`

Reply:
Good catch!

Reply:
Revised to: 
> You must select a subdomain before creating a subdomain aliases